go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/jmoiron/sqlx
```

Comment:
@ebenoist wondering if this go.mod file needs surgery, I'm getting the following error when trying to pull in this new version:

`[cg/sync_active_users][~/dev/feed]$ go get -u "github.com/reverbdotcom/sqlx"
go: downloading github.com/reverbdotcom/sqlx v0.0.0-20210511173003-f5462c8dec5c
go get: github.com/reverbdotcom/sqlx@v0.0.0-20180110223232-8f6781503779 updating to
	github.com/reverbdotcom/sqlx@v0.0.0-20210511173003-f5462c8dec5c: parsing go.mod:
	module declares its path as: github.com/jmoiron/sqlx
	        but was required as: github.com/reverbdotcom/sqlx`